### PR TITLE
os/board/rtl8721csm: Modify printf APIs for BT Init

### PR DESCRIPTION
**Pull request by ziyik**

### CODE REVIEW

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c [hunk 1]
```diff
@@ -279,13 +279,13 @@ boolean _btiqk_lok_8721d(void *dm_void, u32 *u32bt_iqk_x, u32 *u32bt_iqk_y, u32
 	{
 		btiqk_success |= 0x01;
 #if DBG_BT_CALIBRATION 
-        DBG_8195A("the IQK and LOK procedure is done");	/* success != workable */
+        printf("the IQK and LOK procedure is done");	/* success != workable */
```
Comment (jeongchanKim) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c:282]: For `printf`, there is no information where the log is from.
How about adding prefix for easy debugging?
For example, some keyword like `[BT]` or filename and code line.

Comment (sunghan-chang) [inline os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c:282]: Need line ending at the end of the message for printf.

Comment (jeongchanKim) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c:282]: You can use dbg also.

**Review by an4967 — COMMENTED** [review]
This works well but please check above comments.

File: os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c (hunk 1 above)
Comment (sunghan-chang) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c:282]: If they are mandatory logs, printf is correct. But if not, dbg is correct.

Comment (ziyik) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c]: Added

Comment (ziyik) [reply os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/rtl8721d_bt_calibration.c]: Done modify and standardize BLE init log

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**